tools/populate-owners/README.md

```diff
@@ -0,0 +1,28 @@
+# Populating `OWNERS` and `OWNERS_ALIASES`
+
+This utility pulls `OWNERS` and `OWNERS_ALIASES` from upstream OpenShift repositories.
+Usage:
+
+```console
+$ go run main.go
+```
+
+Upstream repositories are calculated from `ci-operator/jobs/{organization}/{repository}`.
+For example, the presence of [`ci-operator/jobs/openshift/origin`](../../ci-operator/jobs/openshift/origin) inserts [openshift/origin][] as an upstream repository.
+
+The `HEAD` branch for each upstream repository is pulled to extract its `OWNERS` and `OWNERS_ALIASES`.
+If `OWNERS` is missing, the utility will ignore `OWNERS_ALIASES`, even if it is present upstream.
+
+Once all the upstream content has been fetched, the utility namespaces any colliding upstream aliases.
+Collisions only occur if multiple upstreams define the same alias with different member sets.
+When that happens, the utility replaces the upstream alias with a `{organization}-{repository}-{upstream-alias}`.
+For example, if [openshift/origin][] and [openshift/installer][] both defined an alias for `security` with different member sets, the utility would rename them to `openshift-origin-security` and `openshift-installer-security` respectively.
+
+After namespacing aliases, the utility writes `OWNERS_ALIASES` to the root of this repository.
+If no upstreams define aliases, then the utility removes `OWNER_ALIASES` from the root of this repository.
+
+The utility also iterates through the `ci-operator/jobs/{organization}/{repository}` directories, writing `OWNERS` to reflect the upstream configuration.
+If the upstream did not have an `OWNERS` file, the utility removes the associated `ci-operator/jobs/{organization}/{repository}/OWNERS`.
```

Comment:
> Overwrite is fine but remove seems wrong

Really?  If upstream adds an `OWNERS`, and we slurp it, and then upstream removes their `OWNERS`, I don't think we want to leave the stale upstream content in this repo.  Of course, it would be best if upstream didn't drop their `OWNERS`, but that's up to them.  And if upstream does drop their `OWNERS`, falling back to whoever owns the parent directory seems sane to me.